task_AUD-1/Person.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Consists of information about person,
+ * checks correctness of entered information.
+ */
+public class Person {
+  private String firstName;
+  private String lastName;
+  private int age;
+  public Person(String[] information) {
+    firstName = information[0];
+    lastName = information[1];
+    age = Integer.valueOf(information[2]);
+    verify();
+  }
+  
+  /**
+   * Checks correctness of entered information about person.
+   * @throw IllegalArgumentException if firstname or lastname was entered in wrong format.
+   * @throw NumberFormatException if age is not positive.
+   */
+  private void verify() {
+    if (stringIsNumber(firstName) || stringIsNumber(lastName)) {
+      throw new IllegalArgumentException();
```

Comment:
You can pass description string as argument of IllegalArgumentException constructor i.e. `new IllegalArgumentException("Wrong first name")` to make your exception handling code more descriptive